contrib/debian/copyright

```diff
@@ -71,12 +71,33 @@ Files: build-aux/m4/ax_check_compile_flag.m4
  build-aux/m4/ax_check_preproc_flag.m4
 Copyright: 2008, Guido U. Draheim <guidod@gmx.de>
  2011, Maarten Bosmans <mkbosmans@gmail.com>
-License: GPL-3+ with Autoconf exception
+License: GPL-3+-with-Autoconf-exception
```

Comment:
NACK. This undoes the changes I made in https://github.com/zcash/zcash/pull/5333/commits/a672318b4d9341a2ac3938f2327a3f218ad88ae8, which were specifically to ensure we followed the v1 format of the Debian copyright file. Note that it says "License names are case-insensitive, and may not contain spaces"; here the license name is `GPL-3+`, and the Autoconf exception is applied to the license with spaces (exactly matching the examples given in the v1 specification).

Reply:
Removed that commit.